torch/_dynamo/variables/builder.py

```diff
@@ -2639,6 +2643,15 @@ def get_automatic_dynamic_shapes_mark_as():
         )
 
 
+def get_dynamic_sources() -> set[str]:
```

Comment:
nit: maybe just define put _DYNAMIC_SOURCES on top of this function. since its only ever used by this function